moveit_planners not installing chomp

### ROS Distro

Jazzy

### OS and version

Ubuntu 24.04

### Source or binary build?

Binary

### If binary, which release version?

2.12.1

### Steps to Reproduce

`moveit_planners` in package.xml
`rosdep install ...`
Result: move_group crashes because chomp not found.

`moveit_planners_chomp` to package.xml
`rosdep install ...`
Result: successful startup

`moveit_planners` does install `ros-jazzy-chomp-motion-planner`, but `move_group` also requires `ros-jazzy-moveit-planners-chomp`.

I basically followed the tutorial and substituted in my own robot's details. I am running `demo.launch.py` and get this error:

```
control-1        | [move_group-2] [FATAL] [1736629917.224762011] [move_group.moveit.moveit.ros.planning_pipeline]: Exception while loading planner 'chomp_interface/CHOMPPlanner': According to the loaded plugin descriptions the class chomp_interface/CHOMPPlanner with base class type planning_interface::PlannerManager does not exist. Declared types are  ompl_interface/OMPLPlanner pilz_industrial_motion_planner/CommandPlanner stomp_moveit/StompPlannerAvailable plugins: ompl_interface/OMPLPlanner, pilz_industrial_motion_planner/CommandPlanner, stomp_moveit/StompPlanner
control-1        | [move_group-2] terminate called after throwing an instance of 'pluginlib::LibraryLoadException'
control-1        | [move_group-2]   what():  According to the loaded plugin descriptions the class chomp_interface/CHOMPPlanner with base class type planning_interface::PlannerManager does not exist. Declared types are  ompl_interface/OMPLPlanner pilz_industrial_motion_planner/CommandPlanner stomp_moveit/StompPlanner
control-1        | [move_group-2] Stack trace (most recent call last):
control-1        | [move_group-2] #16   Object "/usr/lib/aarch64-linux-gnu/ld-linux-aarch64.so.1", at 0xffffffffffffffff, in 
control-1        | [move_group-2] #15   Object "/opt/ros/jazzy/lib/moveit_ros_move_group/move_group", at 0x5555c2b61d6f, in _start
control-1        | [move_group-2] #14   Object "/usr/lib/aarch64-linux-gnu/libc.so.6", at 0x7fff4b5c8597, in __libc_start_main
control-1        | [move_group-2] #13   Object "/usr/lib/aarch64-linux-gnu/libc.so.6", at 0x7fff4b5c84c3, in 
control-1        | [move_group-2] #12   Object "/opt/ros/jazzy/lib/moveit_ros_move_group/move_group", at 0x5555c2b60743, in main
control-1        | [move_group-2] #11   Object "/opt/ros/jazzy/lib/libmoveit_cpp.so.2.12.1", at 0x7fff4c01eb13, in moveit_cpp::MoveItCpp::MoveItCpp(std::shared_ptr<rclcpp::Node> const&, moveit_cpp::MoveItCpp::Options const&)
control-1        | [move_group-2] #10   Object "/opt/ros/jazzy/lib/libmoveit_cpp.so.2.12.1", at 0x7fff4c01a1b7, in moveit_cpp::MoveItCpp::loadPlanningPipelines(moveit_cpp::MoveItCpp::PlanningPipelineOptions const&)
control-1        | [move_group-2] #9    Object "/opt/ros/jazzy/lib/libmoveit_planning_pipeline_interfaces.so.2.12.1", at 0x7fff4b27376b, in moveit::planning_pipeline_interfaces::createPlanningPipelineMap(std::vector<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::allocator<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > > const&, std::shared_ptr<moveit::core::RobotModel const> const&, std::shared_ptr<rclcpp::Node> const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)
control-1        | [move_group-2] #8    Object "/opt/ros/jazzy/lib/libmoveit_planning_pipeline.so.2.12.1", at 0x7fff4b116f93, in planning_pipeline::PlanningPipeline::PlanningPipeline(std::shared_ptr<moveit::core::RobotModel const> const&, std::shared_ptr<rclcpp::Node> const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)
control-1        | [move_group-2] #7    Object "/opt/ros/jazzy/lib/libmoveit_planning_pipeline.so.2.12.1", at 0x7fff4b116883, in planning_pipeline::PlanningPipeline::configure()
control-1        | [move_group-2] #6    Object "/usr/lib/aarch64-linux-gnu/libstdc++.so.6.0.33", at 0x7fff4b85a983, in __cxa_rethrow
control-1        | [move_group-2] #5    Object "/usr/lib/aarch64-linux-gnu/libstdc++.so.6.0.33", at 0x7fff4b850e03, in std::terminate()
control-1        | [move_group-2] #4    Object "/usr/lib/aarch64-linux-gnu/libstdc++.so.6.0.33", at 0x7fff4b85a56f, in 
control-1        | [move_group-2] #3    Object "/usr/lib/aarch64-linux-gnu/libstdc++.so.6.0.33", at 0x7fff4b85d4d7, in __gnu_cxx::__verbose_terminate_handler()
control-1        | [move_group-2] #2    Object "/usr/lib/aarch64-linux-gnu/libc.so.6", at 0x7fff4b5c7dff, in abort
control-1        | [move_group-2] #1    Object "/usr/lib/aarch64-linux-gnu/libc.so.6", at 0x7fff4b5dcb3b, in raise
control-1        | [move_group-2] #0    Object "/usr/lib/aarch64-linux-gnu/libc.so.6", at 0x7fff4b627628, in 
control-1        | [move_group-2] Aborted (Signal sent by tkill() 69 0)
control-1        | [ERROR] [move_group-2]: process has died [pid 69, exit code -6, cmd '/opt/ros/jazzy/lib/moveit_ros_move_group/move_group --ros-args --params-file /tmp/launch_params_rc0z644s --params-file /tmp/launch_params_l2bygw0f'].
```

--- a/moveit_planners/moveit_planners/package.xml
+++ b/moveit_planners/moveit_planners/package.xml
@@ -19,7 +19,7 @@
 
   <buildtool_depend>ament_cmake</buildtool_depend>
 
-  <exec_depend>chomp_motion_planner</exec_depend>
+  <exec_depend>moveit_planners_chomp</exec_depend>
   <exec_depend>moveit_planners_ompl</exec_depend>
   <exec_depend>moveit_planners_stomp</exec_depend>
   <exec_depend>pilz_industrial_motion_planner</exec_depend>

